aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -198,7 +198,7 @@ protected boolean isDBClusterDeleted(
                             ADD_ASSOC_ROLES_ERROR_RULE_SET
                     ))
                     .success();
-            if (!progressEvent.isSuccess()) {
+            if (progressEvent.isFailed()) {
                 return progressEvent;
```

Comment:
I dont think this will lead to the correct result. 

`!progressEvent.isSuccess()` means that event it either failed or in progress. If it's failed we need to return to propagate to the customer. In case it's still in progress but lambda timeout is up, we do need to propagate the intermediate result back to the orchestrator so it will reinvoke and continue stabilization